batch/batch/batch.py

```diff
@@ -56,7 +64,8 @@ def _time_msecs_str(t):
         'time_completed': time_completed,
         'duration': duration,
         'msec_mcpu': record['msec_mcpu'],
-        'cost': coalesce(record.get('cost'), 0),
+        'cost': coalesce(record['cost'], 0),
```

Comment:
This accidentally(?) dropped your change to this line from abd4476942:
```suggestion
        'cost': coalesce(record.get('cost'), 0),
```